docs/reference/how-to/knn-search.asciidoc

```diff
@@ -128,7 +128,7 @@ is often enough. To allow the full indexing buffer to be used, you should also
 increase the limit <<index-modules-translog,`index.translog.flush_threshold_size`>>.
 
 [discrete]
-=== Avoid heavy indexing during searches
+=== Avoid heavy indexing during times of heavy search throughput
```

Comment:
I think you also need to soften the language on line 140 to match this sentiment.